### PR TITLE
Update seats enum

### PR DESCRIPTION
**Pull request by AGraber**
Seats in serverside now start from 0 (clientside is still -1)

### CODE REVIEW

File: enums.d.ts [hunk 1]
```diff
@@ -129,10 +129,10 @@ declare namespace RageEnums {
...
 
 	const enum VehicleSeat {
-		DRIVER = -1,
-		PASSENGER_1 = 0,
-		PASSENGER_2 = 1,
-		PASSENGER_3 = 2
+		DRIVER = 0,
+		PASSENGER_1 = 1,
+		PASSENGER_2 = 2,
+		PASSENGER_3 = 3
```
Comment (pixlcrashr) [inline enums.d.ts]: Aren't there more than 4 seats (including the driver)?

**Review by pixlcrashr — CHANGES_REQUESTED** [review]

File: enums.d.ts (hunk 1 above)
Comment (pixlcrashr) [reply enums.d.ts]: By the way: You can write the enum short by writing this:
```ts
const enum Test {
    DRIVER, // would be equivalent to 0
    PASSENGER_1, // would be equivalent to 1
    PASSENGER_2 // and so on
}
```

Comment (AGraber) [reply enums.d.ts]: Well, passenger seats always vary for each vehicle and some have 6 or 8 (not sure which is the max though). What do you recommend?

Comment (pixlcrashr) [reply enums.d.ts]: Current maximum amount is 16. So 16 passenger seats.

**Review by pixlcrashr — APPROVED** [review]

**Pull request merged**